src/js/components/agency/visualizations/StatusOfFundsChart.jsx

```diff
@@ -170,6 +170,19 @@ const StatusOfFundsChart = ({
 
         return viewHeight * 1.06;
     };
+    const chartLevelText = () => {
+        if (level === 0) {
+            return <><hr /><div className="tooltip__text-note">Click a sub-component to view <br />Federal Accounts</div></>;
+        } else if (level === 1) {
+            return <><hr /><div className="tooltip__text-note">Click a Federal Account to view <br />Treasury Accounts</div></>;
+        } else if (level === 2) {
+            return <><hr /><div className="tooltip__text-note">Click a Treasury Account to view <br />Program Activities or Object Classes</div></>;
+        }
+        return null;
+    };
+    const tooltipHeight = level === 1 ? 280 : 230;
```

Comment:
do these need to say === 1 or >=1?

Reply:
I find that the longest titles are in level 1. and drop back down to normal after 1. So I did ===1.